### DISCUSSION TITLE
Criei um guia de CSS

### DISCUSSION THREAD

**Discussion by Hai-San** [frontendbr/forum#2286] [Mostre seu trabalho]
Fala galerinha.
Eu faço parte de um grupo seleto de DEVs que adora o CSS kkkkk e faz um tempo que eu já estou trabalhando neste projetinho. Agora que já tem bastante conteúdo eu vou deixar aqui pra quem quiser dar uma olhada. 

Em resumo é um guia pra quem curte escrever CSS puro mas não sabe uma forma legal de organizar para ficar fácil de manter e ser escalável.

Link pro [github](https://github.com/Hai-San/orcss)

Quem ter alguma dica já deixa lá na issues :D

Valeuu \o/

**Comment by nazarepiedady** [comment]
É uma boa iniciativa, as filosofias sobre organização de código não são mágicas,
o que abre um caminho para abordagens mais criativas e quem sabe mais inteligíveis.
